tests/test_command.py

```diff
@@ -198,6 +198,25 @@ def test_history_indicate_current(self):
         )
 
 
+class MutationTest(TestBase):
+    @classmethod
+    def setup_class(cls):
+        cls.env = staging_env()
+        cls.cfg = _sqlite_testing_config()
+
+    @classmethod
+    def teardown_class(cls):
+        clear_staging_env()
+
+    def test_current(self):
+        command.current(self.cfg)
+
+        url = self.cfg.get_main_option("sqlalchemy.url")
+        engine = create_engine(url)
```

Comment:
here you should probably use `self.bind`

Reply:
See 0e10855, made it it similar to `LiveStampTest`.

Not sure if we want SQLite only, though.

Reply:
it should be fine only in sqlite